src/core/components/table/SimpleTable.vue

```diff
@@ -1,17 +1,19 @@
 <template>
   <div class="relative-position table-spinner-container">
-    <q-table v-if="!loading" :data="data" :columns="columns" :pagination="pagination" binary-state-sort
-      flat :separator="separator" :selection="selection" :row-key="rowKey" v-on="$listeners" :selected="selected"
-      :visible-columns="formattedVisibleColumns" :hide-bottom="shouldHideBottom"
-      :class="['q-pa-sm large-table sticky-header', isClientInterface ? 'client-header' : 'vendor-header']">
+    <q-table v-if="!loading" flat :columns="columns" :pagination="pagination" binary-state-sort :selection="selection"
+      :rows="data" :separator="separator" :selected="selected" @update:expanded="$emit('update:expanded', $event)"
+      :visible-columns="formattedVisibleColumns" :hide-bottom="shouldHideBottom" @row-click="$emit('row-click', $event)"
+      @update:pagination="$emit('update:pagination')" @update:selected="$emit('update:selected', $event)"
+      :class="['q-pa-sm large-table sticky-header', isClientInterface ? 'client-header' : 'vendor-header']"
+      :row-key="rowKey">
       <template #header="props">
         <slot name="header" :props="props">
           <q-tr :props="props">
             <q-th v-for="col in props.cols" :key="col.name" :props="props" :style="col.style">{{ col.label }}</q-th>
           </q-tr>
         </slot>
       </template>
-      <template v-if="$scopedSlots['top-row']" #top-row="props">
+      <template v-if="$slots['top-row']" #top-row="props">
```

Comment:
Pourquoi ce changement ?

Reply:
pour ca  : https://quasar.dev/start/upgrade-guide#vue-3-and-scoped-slots :)